### DISCUSSION TITLE
Resultados do Survey 2018

### DISCUSSION THREAD

**Discussion by felipefialho** [frontendbr/forum#1286] [Discussão]
Fala pessoal, como estão?

Semana passada terminei o [Survey 2018](https://survey.frontendbr.com.br/), e dá pra ter uma noção do atual cenário frontender no Brasil.

Fiz uma compilação mais quantitativa do que qualitativa, até porque não tive muito tempo para trabalhar no desenvolvimento. 

Para uma analise completa, todas as respostas da pesquisa estão no [Google Sheets](https://docs.google.com/spreadsheets/d/1EjNF3tH0gk5ew0SKrSdbJZqV9jxGtSXf9EXQzL8qLH8/edit?usp=sharing).

Criei essa issue para conversar sobre a pesquisa 🍻 

- O que acharam dos resultados? 
- Salários estão dentro do esperado?
- Em geral, acharam bom ou ruim?

Nóis

**Comment by mksanti** [comment]
> Salários estão dentro do esperado?

Por favor a galera que ganha acima de 15k dê o caminho das pedras de ouro.

**Comment by didiraja** [comment]
Achei os resultados bem condizentes com minha realidade aqui no RJ.

Sei que não é culpa da galera daqui, mas ainda fico incomodado com a amostragem.

**Comment by allansduarte** [comment]
Em geral o survey vem evoluindo, isso é muito bom, pois assim podemos ver o estado do front end no Brasil. Espero poder participar do próximo survey.

**Comment by thismarcoantonio** [comment]
Muito interessante, embora algumas coisas nessa tabela me deixaram triste. Mas bem, cada um julga quanto deve pagar e quanto deve receber.
